code/src/components/TodoList.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { CgClose } from "react-icons/cg";
+
+import todos from "../reducers/todos";
+
+import EmptyList from "../assets/emptyList.png";
+import SubHeader from "./SubHeader";
+
+import {
+  MainWrapper,
+  ItemList,
+  Paragraph,
+  Title,
+  CompleteTask,
+  SendButton,
+  ListWapper,
+  ParagraphWrapper,
+  EmptyListImage
+} from "../styled/StyledComponent";
+
+const TodoList = () => {
+  const items = useSelector((store) => store.todos.items);
+  const dispatch = useDispatch();
+  const [removeItems, setRemoveItems] = useState(false);
+
+  const dateConverter = `
+  ${new Date().toLocaleString("default", {
+    weekday: "long",
+    day: "2-digit",
+    month: "long"
```

Comment:
Cool that you coded your own date. I am so lazy and used moment instead 💯   Very inspiring!